### DISCUSSION TITLE
code-server with home-assistant.io

### DISCUSSION THREAD

**Discussion by bpmct** [coder/code-server#2078] [Show and tell]
There is a Hass.io addon for running code-server. Pretty cool, @frenck. 

https://github.com/hassio-addons/addon-vscode

**Comment by bpmct** [comment]
We currently support armv7 but are not building releases (#1337). I wonder if there would be an easy way to support this now. ..

**Reply by sr229** [reply]
We would need to build using ARMv7 libraries. I don't think `LD_PRELOAD` would cut it since Node.js ignores that I believe.
